pages/common/git-summary.md

```diff
@@ -0,0 +1,17 @@
+# git summary
+
+> Display information about a Git repository.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-summary>.
+
+- Display data about a Git repository:
+
+`git summary`
+
+- Display data about a Git repository since a commit-ish:
+
+`git summary {{commit|branch_name|tag_name}}`
```

Comment:
```suggestion
`git summary {{commit_hash|branch_name|tag_name}}`
```

Reply:
`commit` has 23 examples, `commit_hash` 11. :man_shrugging:

Reply:
But what are the contexts of each. 

Hash works better here IMO and also makes it look better since the others all have underscores.

Reply:
I think the only way to reference a commit is the hash. So `commit` can't be another thing than the `hash`.
I know a tag is a named commit, but that's mentioned in this example.

Reply:
Since TLDR is aimed for beginners as well, I don't think most know that a tag is just a glorified commit.

Reply:
My `approve` and the :man_shrugging: at the same time meaning that I think both are good, I don't have an strong opinion.

Reply:
@bl-ue @marchersimon not a big deal this suggestion however best to have everyones opinion.

Reply:
I think I'd prefer just `commit` here, though as @navarroaxel says either would work. In this case I think it's better to be concise, as it reads fine and is less to parse for the reader?

Reply:
Suits me. Resolved.

Reply:
Leaving open in case others want to discuss a little

Reply:
I'd go with `commit_hash`, since I think many beginners would be confused by `commit`. I originally thought about a commit as the changes it made to files. `commit_hash` is not wrong in any way and it's clear.

Reply:
I still like hash.
sbrl likes commit
navarroaxel doesn't mind
marchersimon likes hash
@bl-ue and @SethFalco could post their opinions to settle this.

Reply:
> A special notation "**<commit1>..<commit2>**" can be used as a short-hand for "**^<commit1> <commit2>**". For example, either of the following may be used interchangeably:
> ```
> $ git log origin..HEAD
> $ git log HEAD ^origin
> ```
> \- https://www.git-scm.com/docs/git-log

I recognize this isn't `git log`, but if it's commit resolutions are similar to it, then I think just `commit` on it's own is best. `origin`, `HEAD`, `^origin`, `HEAD~2`, etc are all valid ways to reference a commit but are not a hash.

Reply:
I opted to just install `git-extras` and test it.

I can execute:
```bash
git summary HEAD~6750
```

`HEAD~6750` means I want to reference the commit 6750 commits before the current HEAD.

The response:
```bash

 project  : tldr
 repo age : 8 years
 active   : 9 days
 commits  : 36
 authors  : 
    11  Romain Prieto        30.6%
     5  rprieto              13.9%
     4  Pranav Raja          11.1%
     3  John Wu              8.3%
     3  Rory Crispin         8.3%
     2  Shrayas Rajagopal    5.6%
     2  marekhrabe           5.6%
     1  Benedikt Ritter      2.8%
     1  Camilla Krag Jensen  2.8%
     1  Erwann Mest          2.8%
     1  Ruy Adorno           2.8%
     1  Zlatan Vasović       2.8%
     1  andrewboerema        2.8%
```

So yeah, the commit resolution is the same, so there are many more ways other than the hash to reference a commit.

Reply:
Git extras is like tldr of git

Would recommend keeping it

Reply:
Thanks seth!